Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -218,8 +222,9 @@ theorem commutes : ∀ r : R, e (algebraMap R A₁ r) = algebraMap R A₂ r :=
 #align alg_equiv.commutes AlgEquiv.commutes
 
 -- @[simp] -- Porting note (#10618): simp can prove this
-theorem map_smul (r : R) (x : A₁) : e (r • x) = r • e x := by
-  simp only [Algebra.smul_def, map_mul, commutes]
+@[deprecated _root_.map_smul (since := "2024-06-20")]
+theorem map_smul (r : R) (x : A₁) : e (r • x) = r • e x :=
```

Comment:
What do you think of making this `protected` at the same time? Will save us from removing a few `_root_` qualifiers.

```suggestion
protected theorem map_smul (r : R) (x : A₁) : e (r • x) = r • e x :=
```